tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Shared/ContextPropagation.cs

```diff
@@ -28,16 +28,18 @@ private static void Inject(PropagationContext context, IDictionary messageAttrib
             {
                 var encodedBytes = PathwayContextEncoder.Encode(context.SpanContext.PathwayContext.Value);
                 var base64EncodedContext = Convert.ToBase64String(encodedBytes);
-                sb.Append($"\"{DataStreamsPropagationHeaders.PropagationKeyBase64}\":\"")
-                .Append(base64EncodedContext)
-                .Append('"');
+                const string ddPathwayCtxBase64 = $"\"{DataStreamsPropagationHeaders.PropagationKeyBase64}\":\"";
```

Comment:
This declaration/extraction wasn't necessary, just proves (to me) that it's definitely a `const` 😄